README.md

```diff
@@ -159,6 +159,21 @@ By default, requests to files with the following extensions will be ignored by t
 
 If you want to specify this yourself, add `IgnoredResourceExtensions` to the configuration.
 
+### Specifying ignored URLs
+
+If certain URLs should be ignored, you can use the `IgnoreSuggestionsUrlRegexPattern` option:
+
+```
+services.AddNotFoundHandler(o =>
+{
+    o.IgnoreSuggestionsUrlRegexPattern = @"^(https?:\/\/[^\/]+)?\/(api|episerverapi|globalassets|siteassets)";
+    // The first group in this pattern is common if LogWithHostname is set to true
```

Comment:
Not sure I understand this comment

Reply:
I mean this should be in place if LogWithHostname is set to true in your regex pattern